src/main/target/STM32_UNIFIED/target.h

```diff
@@ -194,6 +236,8 @@
 #define USE_SDCARD_SPI
 #define USE_SDCARD_SDIO
 
+#endif
```

Comment:
There are probably better things to remove than SD card support - will onboard VTX, transponder, VTX be enough?

Reply:
oh yes, removing onboard VTX is a great idea. I can still see some ppl using a transponder and VTX control on the F411 board.

Well my English is not good enough - do you mean that you don't want to remove the SD card support?

Reply:
oh sorry misunderstood, you mean I already delete those features?

Reply:
Well this is the building info after cutting 
- RTC6705
- RTC6705_SOFTSPI
- TRANSPONDER
and add SDcard support back in

![image](https://user-images.githubusercontent.com/31283897/116033178-882d3700-a693-11eb-818c-fef5ac659a8b.png)